node/lib/task.ts

```diff
@@ -171,9 +200,10 @@ export function setResourcePath(path: string): void {
         resourceFile = path;
         debug('set resource file to: ' + resourceFile);
 
-        var locStrs = loadLocStrings(resourceFile);
+        resourceCulture = getVariable('system.culture') || resourceCulture;
```

Comment:
It's initialized only if TASKLIB_INPROC_UNITS. I have no idea what that is, so I'll take your word :)